docs/get_started/plan_deployment/traditional_deployment/plan_multiple_profile.md

```diff
@@ -1,47 +1,47 @@
+Starting with Portal Version 7, you can create extra profiles in addition to the default profile. Extra proviles are useful in the following scenarios:
 
--   Reuse a single installation for multiple independent portal instances, such as for various test or development efforts.
--   Recover from a configuration problem by deleting the current profile and re-creating it without having to reinstall the product.
+-   Reuse a single installation for multiple, independent portal instances, such as for various test or development efforts.
+-   Recover from a configuration problem by deleting the current profile and re-creating it without reinstalling the product.
 -   Update a Deployment Manager profile to handle portal servers and thus avoid all the manual preparation steps.
 
-The following profile types can be generated:
+You can generate the following profile types:
 
 -   **portal.default**
 
-    This profile holds a stand-alone portal server in the captured configuration. It might be used as a stand-alone server or after the federation as a base for a portal cluster.
+    This profile holds a stand-alone portal server in the captured configuration. The server might be used as a stand-alone server or, after federation, as a base for a portal cluster.
 
 -   **managed.portal**
 
-    This profile is enhanced with the portal run time environment but does not contain any servers. The main purpose of this profile is to use it as a run time environment for extra cluster members. After the federation of this profile, the standard portal tasks can be used to create a portal server cluster member from an existing cluster template.
+    This profile is enhanced with the portal runtime environment but does not contain any servers. The main purpose of this profile is to use it as a runtime environment for extra cluster members. After the federation of this profile, you can complete standard portal tasks to create a portal server cluster member from an existing cluster template.
 
 -   **management.portal.augment**
 
-    This profile is created by augmenting a standard Deployment Manager profile. The resulting profile holds a deployment manager that is prepared for use with portal. It can be used to federate the other portal profiles immediately without the need for the additional manual steps. Because the deployment manager is often on different hardware, there is the possibility to move this augmentation template to another installation and run it there.
+    You create this profile by augmenting a standard Deployment Manager profile. The resulting profile holds a deployment manager that is prepared to use with a portal. You can used the portal to federate the other portal profiles immediately without requiring additional manual steps. Because the deployment manager is often on different hardware, you can move this augmentation template to another installation and run it there.
```

Comment:
Kindly change 
"You create this profile by augmenting a standard Deployment Manager profile. The resulting profile holds a deployment manager that is prepared to use with a portal. You can use the portal to federate the other portal profiles immediately without additional manual steps. Because the deployment manager is often on different hardware, you can move this augmentation template to another installation and run it there."